### PR TITLE
LPS-163243

### PR DESCRIPTION
**Pull request by sofiamendozalr**
- LPS-163243 Add execute-all-pending-upgrades-from-gogo-shell to build-test-db-upgrade-client.xml
- LPS-163243 Add ExecuteAllPendingUpgradesFromGogoShell testcase

Issue link: https://issues.liferay.com/browse/LPS-163243

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/upgrades/DatabaseUpgradeClient.testcase [hunk 1]
```diff
@@ -59,4 +59,16 @@ definition {
 			value1 = "check-upgrade-client-zip-content");
 	}
 
+	@description = "The user can execute the upgrade:executeAll command on Gogo shell to execute all remaining upgrade processes"
+	@priority = "3"
+	test ExecuteAllPendingUpgradesFromGogoShell {
```
Comment (susanchen3) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/upgrades/DatabaseUpgradeClient.testcase:64]: Add 
```
property skip.get.testcase.database.properties = "true";
property skip.upgrade-legacy-database = "true";
```

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/upgrades/PortalDatabaseValidationUpgrade.testcase [hunk 2]
```diff
@@ -9,8 +9,7 @@ definition {
 
 	@priority = "4"
 	test CompareUpgradedAndFreshDatabaseSchemas7310 {
-		property data.archive.type = "data-archive-portal";
```
Comment (susanchen3) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/upgrades/PortalDatabaseValidationUpgrade.testcase:12]: Why are we removing this? :thinking:

**Review by susanchen3 — CHANGES_REQUESTED** [review]
Can you update your liferay portal? I changed the location of `DatabaseUpgradeClient.testcase` in a recent PR and it doesn't seem to be updated for you. Hopefully this should fix your issues.

This is what it looks like for me locally when I was testing: https://github.com/susanchen3/liferay-portal/pull/105

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/upgrades/PortalDatabaseValidationUpgrade.testcase (hunk 2 above)
Comment (sofiamendozalr) [reply portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/upgrades/PortalDatabaseValidationUpgrade.testcase:12]: I think I deleted it by mistake :(

**Comment by lucasperj** [comment]
Hey @sofiamendozalr , can you organize your commits, rebase your branch and also send a commit with the expect file please? Thank you so much! :smile:

**Comment by sofiamendozalr** [comment]
@vicnate5

**Comment by vicnate5** [comment]
See https://github.com/vicnate5/liferay-portal/pull/2244

**Comment by vicnate5** [comment]
ci:close

**Pull request closed without merging**